ggml/src/ggml-vulkan/ggml-vulkan.cpp

```diff
@@ -2402,6 +2406,12 @@ static vk_buffer ggml_vk_create_buffer(vk_device& device, size_t size, const std
         nullptr,
     };
 
+    vk::ExternalMemoryBufferCreateInfo external_memory_bci;
+    if (import_ptr) {
+        external_memory_bci.handleTypes = vk::ExternalMemoryHandleTypeFlagBits::eHostAllocationEXT;
+        buffer_create_info.setPNext(&external_memory_bci);
+    }
+
     buf->buffer = device->device.createBuffer(buffer_create_info);
```

Comment:
This `createBuffer()` call now throws on Mac:

https://github.com/ggml-org/llama.cpp/actions/runs/20601782380/job/59168854452?pr=18467

Reply:
This is a MoltenVK bug and is fixed by https://github.com/KhronosGroup/MoltenVK/pull/2622. I suspect this isn't in the latest Vulkan SDK. I guess I'll need to disable this path for MoltenVK...